cpp/INIReader.h

```diff
@@ -42,6 +42,9 @@ class INIReader
     // and valid false values are "false", "no", "off", "0" (not case sensitive).
     bool GetBoolean(const std::string& section, const std::string& name, bool default_value) const;
 
+    // Checks whether the key at the given section exists
```

Comment:
Let's make this comment a bit more accurate:

    // Return true if a value exists with the given section and field names.